srtcore/packetfilter.cpp

```diff
@@ -232,7 +232,9 @@ void PacketFilter::globalInit()
     // as builtin. This will disallow users to register
     // external filters with the same name.
 
-    filters["fec"] = new Creator<FECFilterBuiltin>;
+
+    ManagedPtr c = new Creator<FECFilterBuiltin>;
+    filters["fec"] = c;
```

Comment:
What is the purpose?

Reply:
That was during the development I had a problem with the value of `ManagedPtr` in this operation. Now probably redundant.

Reply:
Please delete.

Reply:
Well, I changed my mind :D